static/js/components/PersonalTab.js

```diff
@@ -20,7 +20,7 @@ class PersonalTab extends React.Component {
     return <div>
       <Grid className="profile-splash">
         <Cell col={12}>
-          Please tell us more about yourself so you can participate in the MicroMaster’s
+          Please tell us more about yourself so you can participate in the MicroMasters
           community and qualify for your MicroMaster’s credential.
```

Comment:
Yes, I talked with @pdpinch earlier and MicroMaster's degree is the only case where there should be an apostrophe